### ISSUE TITLE
Document whether/how to use for public google groups (@googlegroups.com)

### ISSUE THREAD

**Issue by reece** [terraform-google-modules/terraform-google-group#46]
### TL;DR

I'd like to use terraform to manage several public google groups for an open source project. It's unclear whether or how this module might be used for groups in the googlegroups.com domain. 

### Terraform Resources

terraform-google-modules/group/google

### Detailed design

I have a terraform repo that currently manages aws and github resources for a small number of users in an open source project (https://biocommons.org). To continue this direction, I would like to now add 6 public google groups now (e.g., [biocommons-announce](https://groups.google.com/g/biocommons-announce)).

The goal is ultimately to be able to add and remove people, especially those with privileged project roles, using IAC rather than a UI.
